src/minkowski.jl

```diff
@@ -0,0 +1,75 @@
+"""
+    Minkowski(order::Real)
+
+Minkowski distance of order p
+
+```math
+D(X,Y) = (\\sum_{i=1}^n \\left|X_i - Y_i\\right|^p)^{1/p}
+```
+
+"""
+struct Minkowski <: ImageDistances.ImageMetric
+    order::Real
+end
+
+"""
+    minkowski_p(d::Minkowski, imgA, imgB)
+
+minkowski distance to p-th power
+
+See also: [`minkowski`](@ref)
+"""
+function minkowski_p(d::Minkowski, imgA::AbstractArray, imgB::AbstractArray)::Real
```

Comment:
Actually I don't know what a proper name for this would be due to my poor English.

Reply:
Thanks @johnnychen94 , I would just call it `minkowski` without the `_p` suffix.